deployment/network-operator/charts/sriov-network-operator/crds/sriovnetwork.openshift.io_sriovnetworknodepolicies_crd.yaml

```diff
@@ -73,7 +79,11 @@ spec:
                 properties:
                   deviceID:
                     description: The device hex code of SR-IoV device. Allowed value
-                      "158b", "1015", "1017".
+                      "0d58", "1572", "158b", "1013", "1015", "1017", "101b".
```

Comment:
I think we should remove the intel devices https://pci-ids.ucw.cz/read/PC/8086

Reply:
@moshe010 should we add 1013 ConnectX-4?

Reply:
thats just the description string in the CRD.

this is a direct copy from sriov-network-operator.

if we want network-operator to define a different set of CRDs that sriov-network-operator will work with then:

1. they need to be fully compatible with sriov-network-operator different controllers so they operate as expected
2. maintain these crds in their own folder in the project, these should be then mirrored to the chart we maintain under `deployment`

in addition we may also want to explicitly define some config maps sriov-network-operator uses to customize its controllers which are created by default when its deployed.